src/main/java/org/fundacionjala/pivotal/pages/ConfirmAction.java

```diff
@@ -32,30 +32,30 @@
     /**
      * Delete through link option.
      */
-    public void clickOnDeleteProjectLink() {
+    public void clickOnDeleteWorkspaceProjectLink() {
         action.scrollToElement(deleteLink);
         action.click(deleteLink);
     }
 
     /**
      * Confirm deletion.
      */
-    public void clickOnDeleteButton() {
+    public void clickOnConfirmWorkspaceProjectDeleteButton() {
```

Comment:
please restore before name

Reply:
change to `clickOnDeleteButton` on next commit